sdk/core/Azure.Core.TestFramework/src/RecordedTestAttribute.cs

```diff
@@ -83,7 +83,7 @@ private static bool IsTestFailedWithRecordingMismatch(TestExecutionContext conte
             }
         }
 
-        private static void SetRecordMode(RecordedTestBase fixture, RecordedTestMode mode)
+        private static void SetRecordMode(ClientTestBase fixture, RecordedTestMode mode)
```

Comment:
I don't think we should have `Mode` on the `ClientTestBase `

Reply:
This was so we could remove RecordedTestBase{TEviroment}.cs. If we have reason to keep that class, can remove this.

Reply:
Yes, being able to reference a recorded test class without knowing the generic argument is the reason to keep the `RecordedTestBase` and avoid this change.